### PR TITLE
KEH-384 | Team Overview

### PR DESCRIPTION
**Pull request by TotalDwarf03**
Added section for team overview comparing team acceptance rates.

This PR depend on #38.

### CODE REVIEW

File: lambda_data_logger/main.py [hunk 1]
```diff
@@ -117,6 +119,45 @@ def handler(event, context):
...
+        response = gh.get(f"/orgs/{org}/teams", params={"per_page": 100, "page": page})
+        teams = response.json()
+        for team in teams:
+            usage_data = gh.get(f"/orgs/{org}/team/{team['name']}/copilot/usage")
+            try:
+                if usage_data.json() != []:
+                    copilot_teams.append(team["name"])
+            except Exception as error:
+                # If Exception, then the team does not have copilot usage data and can be skipped
+                pass
```
Comment (evisos-ons) [inline lambda_data_logger/main.py:146]: Same comment as the Team Usage Dropdown PR about the nested loop and try/except comment.

File: src/pages/team_usage.py [hunk 2]
```diff
@@ -241,6 +235,74 @@ def get_team_seats(team):
...
+        elif acceptance_rate >= 20 and acceptance_rate < 30:
+            acceptnace_group = "20-30%"
+        elif acceptance_rate >= 30 and acceptance_rate < 40:
+            acceptnace_group = "30-40%"
+        elif acceptance_rate >= 40 and acceptance_rate < 50:
+            acceptnace_group = "40-50%"
+        elif acceptance_rate >= 50 and acceptance_rate < 60:
+            acceptnace_group = "50-60%"
+        else:
+            acceptnace_group = "60%+"
```
Comment (evisos-ons) [inline src/pages/team_usage.py:299]: 'acceptnace' should be 'acceptance'?

File: src/pages/team_usage.py [hunk 3]
```diff
@@ -241,6 +235,74 @@ def get_team_seats(team):
...
+
+        df_team_acceptance["Acceptance Group"][index] = acceptnace_group
```
Comment (evisos-ons) [inline src/pages/team_usage.py:301]: Same comment on spelling.

File: lambda_data_logger/main.py [hunk 4]
```diff
@@ -117,6 +119,45 @@ def handler(event, context):
...
+    except KeyError:
+        last_page = 1
+
+    for page in range(1, last_page + 1):
+        response = gh.get(f"/orgs/{org}/teams", params={"per_page": 100, "page": page})
+        teams = response.json()
+        for team in teams:
+            usage_data = gh.get(f"/orgs/{org}/team/{team['name']}/copilot/usage")
+            try:
+                if usage_data.json() != []:
```
Comment (delterr) [inline lambda_data_logger/main.py:142]: Minor - might be easier to read if it's `len(usage_data.json()) == 0`

**Review by delterr — APPROVED** [review]
Looks good

**Review by evisos-ons — CHANGES_REQUESTED** [review]
Fahad clearly wants carnage if this tool becomes available to others lol.

I added a comment on the spelling, other than that it looks good.

File: lambda_data_logger/main.py (hunk 1 above)
Comment (TotalDwarf03) [reply lambda_data_logger/main.py]: Done

**Review by evisos-ons — APPROVED** [review]
Nice  👍

**Pull request merged**